Gemfile.lock

```diff
@@ -196,9 +196,8 @@ GEM
       mime-types (~> 3.0)
       representable (~> 3.0)
       retriable (>= 2.0, < 4.0)
-    google-protobuf (3.6.1)
-    google-protobuf (3.6.1-universal-darwin)
```

Comment:
Why was this changed?

Reply:
We manually corrected this after the merge in https://github.com/cloudfoundry/cloud_controller_ng/commit/924c904d9a7ca948407a102685ba215f3a5eff20.